src/testTools.js

```diff
@@ -16,3 +17,23 @@ export function createMouseEvent(eventName) {
         view: window
     });
 }
+
+export const spyOnBuiltins = (functionNames, object = window) => {
+    functionNames.forEach(functionName => {
```

Comment:
Nice!

Reply:
Agreed, the helper functions will be nice!